contents/data-display/avatars/avatars-with-hover-motion/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { Avatar, AvatarGroup, Motion, Tooltip } from "@yamada-ui/react"
+import type { FC } from "react"
+import { users } from "./users"
+
+const AvatarWithHoverFramer: FC = () => {
+  return (
+    <AvatarGroup
+      as={Motion}
+      whileHover={{
+        maxWidth: "450px",
+      }}
+      maxW="md"
+      w="full"
+      gap="-20"
+      justifyContent="space-between"
+      display="flex"
+    >
+      {users.map((user, index) => (
+        <Motion
+          key={user.id}
+          position="relative"
+          whileHover={{
```

Comment:
@taroj1205
When the scale is identical for both hover and tap states, it might be difficult for users to distinguish between these interactions.
 I suggest using different scale values for each state to provide clearer visual feedback. For example:
```
CopywhileHover={{
  scale: 1.3,
  zIndex: 1,
}}
whileTap={{
  scale: 0.8,
  zIndex: 1,
}}
```